doc/en/components/grids/pivot-grid/pivot-grid-features.md

```diff
@@ -0,0 +1,160 @@
+
+A different size can also be set initially with the `width` property available in the dimension definition:
+
+```typescript
+public pivotConfigHierarchy: IPivotConfiguration = {
+    rows: [
+        {
+            memberName: 'Country',
+            enabled: true,
+            width: '400px'
+        }
+    ]
+}
+```
+
+## Dimensions selection
+
+The Pivot Grid supports single selection which is enabled just like in the base grid. For example:
+
+```html
+<{ComponentSelector} #grid1 [rowSelection]="'single'" [data]="data" [pivotConfiguration]="pivotConfigHierarchy">
+</{ComponentSelector}>
+```
+
+In case there are multiple row or column dimensions which would create groups that span multiple rows/columns, selection is applied to all cells that belong to the selected group.
+
+## Super Compact Mode
+The `{ComponentName}` component provides a `superCompactMode` `@Input`. It is suitable for cases that require a lot of cells to be present on the screen at once. If enabled the option ignores the `displayDensity` option for the pivot grid. Enabling `superCompactMode` also sets the `displayDensity` input to `compact` for each child component(like `IgxChip`) that does not have the `superCompactMode` option.
+
+```html
+<{ComponentSelector} [superCompactMode]="true"></{ComponentSelector}>
+```
+
+## Additional summary column
+
+When a `column` dimension defines a hierarchy, the pivot grid will render additional summary/total column, which accumulates the aggregations of all of the columns inside the group. When the group is collapsed only the summary column will remain. And when the group is expanded the additional summary column appears at the end of the group.
+
+## Interactions
+
+### Keyboard navigation
+
+Keyboard navigation in `{ComponentName}` works similarly to the one in `IgxGrid`. The pivot grid is split into three areas - `rows`, `columns`, `values`. The areas for `rows` and `columns` are considered headers for the purposes of navigation while the area for `values` is the body.
+The keyboard arrows allow navigating the active element within the current area only.
+
+### Dimensions drag & drop
+The dimensions are represented by chips, which can be dragged & dropped.
+All chips can change their order within their area by drag & drop.
+The chips from `rows`, `column`, `filter`(dimension chips) can be moved from any of those areas to any other and at any place.
+Chips from these areas can not be moved to the `values` area and chips from the `values` area can not be moved to any of the dimension areas.
+
+>[!NOTE]
+>The chips from the Pivot Grid can not be moved to the Pivot Data Selector and items from the Pivot Data Selector can not be moved to the Pivot Grid.
+
+## API References
+* [{ComponentName}Component]({environment:{Platform}ApiUrl}/classes/{ComponentName}component.html)
+* [IgxPivotDataSelectorComponent]({environment:{Platform}ApiUrl}/classes/igxpivotdataselectorcomponent.html)
```

Comment:
do not hard-code URLs for API link, instead just list API members in backticks, e.g.
``` `{PivotGridName}` ```

Reply:
Updated the docs, please review again

Reply:
Also I am not really sure on filteringExpressionTree for dimensions, there is no value parameter exposed, seems kinda incomplete at this point